crest/Assets/Crest/Crest/Scripts/OceanRenderer.cs

```diff
@@ -290,7 +290,7 @@ static void EditorUpdate()
 
             if (!EditorApplication.isPlaying)
             {
-                if (EditorApplication.timeSinceStartup - _lastUpdateEditorTime > 1f / Mathf.Clamp(Instance._editModeFPS, 0.1f, 60f))
+                if (EditorApplication.timeSinceStartup - _lastUpdateEditorTime > 1f / Mathf.Clamp(Instance._editModeFPS, 0.01f, 60f))
```

Comment:
Could pre-pend:
-> `Instance._editModeFPS > 0`?

Also, 60 FPS can be quite a low upper-limit, people who like using the 144Hz monitors at 144Hz are *really* passionate about it. (I know of at least one instance where a game received one really bad review just because they assumed 60FPS would be the max). Maybe 240Hz would be a good upper-bound? (or at least don't clamp it here, so that people with know-how can set the _editModeFPS value themselves without it being auto-clamped).

Reply:
The way the code is now, it ensures that one update will happen, so that the ocean is in a more or less presentable state. I could add more state to ensure this but was tempted to take this lazy route.

good point about higher freqs, but I'd need to check that the editor runs at >60hz, which i'm not sure i can do on this laptop.

Reply:
![image](https://user-images.githubusercontent.com/10348641/82728841-eebf0d80-9cea-11ea-9f57-a549fe89e8be.png)

I don't know if this helps.

Reply:
https://forum.unity.com/threads/how-to-setup-the-editor-to-run-at-144hz.797001/

It seems to be supported at least. It's a niche use-case for sure though. Although with the growth of variable-frame-rate monitors I imagine that more-and-more people might start caring.

Reply:
I feel like people are going to see the slider go up to 144 and then complain to me that the editor is only running at 60hz. would it be ok to leave it at 60 and wait and see if anyone requests the higher rate? i like to keep slider ranges tight where possible so its quick and easy to set it to 60 which is currently the standard.